.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,201 @@
+# How to contribute
+
+## General Rules and Agreements
+
+### Refactor and improve code as you work
+
+It is okay to include small unrelated changes in your branch that improve code quality. We allow this to avoid creating issues/tasks that would take longer to describe than to fix.
+
+Examples of changes that fit this rule:
+
+- Improving a non-descriptive variable's name
+- Moving hardcoded values into constants
+- General reorganization of code
+- Adding comments to describe a piece of code that is hard to understand
+- Adding missing typing
+- Adding a missing test that involves code that you are touching
+
+### Document your public functions
+
+Use [JSDoc](http://usejsdoc.org/) comments for public methods. _(Note: Since we use a type-system defining paremeter types is redundant, so avoid doing that)_
```

Comment:
Since I don't trust flow, I've been defining them anyway (many of the libraries we use aren't typed, particularly web3, so many parameters won't get typed automatically in the docs). Is there any side-effect from doing this?

Reply:
No, only possible kissmatches between types and JSDocs. If typing is not there, then adding the to the JSDoc comments makes perfect sense